src/screens/ForgotPassword/index.tsx

```diff
@@ -73,7 +73,7 @@ const ForgotScreen = () => {
 
   return (
     <ScreenWrapper>
-      <VStack>
+      <VStack px={8}>
```

Comment:
avoid magical numbers. Use constants from `src/styles/theme.ts` instead

Reply:
I created a file to store sizes constants